messages.js

```diff
@@ -40,9 +40,22 @@ Your changes were [merged successfully](https://github.com/hasura/graphql-engine
 Thanks again :hugs:`;
 };
 
+const depGraphHelpMessage = `Available commands:
+\`/dep-graph\`: Display this help message
+
```

Comment:
I guess it is ok to not mention this explicitly. So maybe we can remove these lines.